backend/runners/load_representative_points.py

```diff
@@ -55,7 +55,8 @@ def _insert_service_areas(json_features):
             sql_class=service_area.ServiceArea,
             data=data,
             return_insert_ids=False,
-            unique=False
+            unique=True,
```

Comment:
Does it make sense to have both `unique` and `unique_column`? Or should `if unique_column` determine the behavior of `unique`?